docs/api/saved-objects/export_by_type.asciidoc

```diff
@@ -0,0 +1,28 @@
+[[saved-objects-api-export-by-type]]
+=== Export Objects By Type
+
+experimental[This functionality is *experimental* and may be changed or removed completely in a future release.]
+
+The export saved objects by type API enables you to retrieve a set of Kibana saved objects into a newline delimited JSON format.
```

Comment:
Maybe we should avoid mentioning the format here, since we intend the format to be mostly irrelevant to people. How about:
```suggestion
The export saved objects by type API enables you to retrieve a set of saved objects that can later be imported into Kibana.
```